requirements/ci.txt

```diff
@@ -36,13 +36,13 @@ six==1.16.0
     # via tox
 tomli==2.0.1
     # via tox
-tox==3.26.0
+tox==3.27.0
```

Comment:
No breaking change in new version: https://tox.wiki/en/latest/changelog.html#v3-27-0-2022-10-25